src/libraries/System.ComponentModel.TypeConverter/src/System/Drawing/SizeConverter.cs

```diff
@@ -70,16 +67,16 @@ public override bool CanConvertTo(ITypeDescriptorContext? context, [NotNullWhen(
                     TypeConverter intConverter = TypeDescriptor.GetConverterTrimUnsafe(typeof(int));
 
                     // Note: ConvertToString will raise exception if value cannot be converted.
-                    var args = new string?[]
-                    {
+                    string?[] args =
```

Comment:
`Could this be $"{intConverter.ConvertToString(context, culture, size.Width)}{culture.TextInfo.ListSeparator} {intConverter.ConvertToString(context, culture, size.Height)}"`

to avoid array allocation?